### PR TITLE
fixed SolverManager.solver() description typo

### PR DESCRIPTION
**Pull request by IanShow15**
### Checklist
- [ ] Documentation: PlannerConfiguration-SolverManager description clarification

<details>
<summary>
fixed SolverManager description typo
</summary>
</details>

### CODE REVIEW

**Comment by IanShow15** [comment]
ok to test

**Comment by triceo** [comment]
ok to test.

File: optaplanner-docs/src/main/asciidoc/PlannerConfiguration/PlannerConfiguration-chapter.adoc [hunk 1]
```diff
@@ -1754,7 +1754,7 @@ A `SolverManager` is a facade for one or more `Solver` instances
 to simplify solving planning problems in REST and other enterprise services.
 Its `solve(...)` methods differ from the normal `Solver.solve(...)` method:
```
Comment (triceo) [inline optaplanner-docs/src/main/asciidoc/PlannerConfiguration/PlannerConfiguration-chapter.adoc:1755]:
```suggestion
Unlike the `Solver.solve(...)` method:
```

File: optaplanner-docs/src/main/asciidoc/PlannerConfiguration/PlannerConfiguration-chapter.adoc [hunk 2]
```diff
@@ -1754,7 +1754,7 @@ A `SolverManager` is a facade for one or more `Solver` instances
...
 
-* *`SolverManager.solve(...)` returns immediately*: it schedules a problem for asynchronous solving without blocking the calling thread.
+* *`Solver.solve(...)` returns immediately*: it schedules a problem for asynchronous solving without blocking the calling thread.
```
Comment (triceo) [inline optaplanner-docs/src/main/asciidoc/PlannerConfiguration/PlannerConfiguration-chapter.adoc:1757]:
```suggestion
* *`SolverManager.solve(...)` returns immediately*: it schedules a problem for asynchronous solving without blocking the calling thread.
```

**Review by triceo — CHANGES_REQUESTED** [review]
@IanShow15 Thank you for your contribution.

I have reviewed the doc in question and have figured out what it was supposed to mean.
I am suggesting a clarification that makes the true meaning of the text obvious. 

Please accept my suggestions by clicking on the appropriate buttons. I will merge afterwards.

**Comment by triceo** [comment]
Thank you!

**Pull request merged**